Content.Server/GameObjects/Components/Power/BaseNetConnectorComponent.cs

```diff
@@ -1,3 +1,5 @@
+#nullable enable
+using System.Diagnostics.CodeAnalysis;
```

Comment:
Doesn't seem used?

Reply:
Used for [NotNullWhen] on a TryGet method